### ISSUE TITLE
titleTemplate is not working when titleChunk is undefined

### ISSUE THREAD

**Issue by morugetsm** [nuxt/framework#9196]
### Environment

- Operating System: `Darwin`
- Node Version:     `v19.1.0`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@8.19.3`
- Builder:          `vite`
- User Config:      `modules`, `runtimeConfig`, `vite`
- Runtime Modules:  `@nuxtjs/color-mode@3.1.8`
- Build Modules:    `-`

### Reproduction

use `titleTemplate` without `title`

### Describe the bug

`titleTemplate` in `useHead` is working with these well.

- nuxt.config.ts -> app.head.title
- script -> useHead.title
- template -> <Title/>

<img width="1728" alt="스크린샷 2022-11-20 오후 4 15 44" src="https://user-images.githubusercontent.com/90916540/202890527-2a256b73-5b5c-4bd6-841e-7521f528f753.png">

But when I remove `title`, page title does not change.
After refresh, 'localhost:3000/' is shown rather than default title although `titleTemplate` function is called.

<img width="1728" alt="스크린샷 2022-11-20 오후 4 15 59" src="https://user-images.githubusercontent.com/90916540/202890535-6bee38f5-2d44-4229-9237-2b6a0f56a5f2.png">

I'm not certain this is a bug or not.
If I miss something, How can I print 'Title' that located on third position of optional operator in `titleTemplate`?

**Comment by danielroe** [comment]
cc: @harlan-zw

**Comment by DrPepperG** [comment]
Upgraded from v3.0.0-rc.13 to v3.0.0 and can confirm the same issue.
```typescript
useHead({
    titleTemplate: (title) => { return title ? `${title} | ${$settings.site_name}` : `${$settings.site_name}` }
})
```

**Comment by dosstx** [comment]
same  here

**Comment by harlan-zw** [comment]
Thanks for reporting this issue. 

Have isolated the problem as described, if you don't manually reset the head with `null`, it won't update. This is now fixed with @vueuse/head 1.0.17.

To upgrade either delete lock file / node_modules and re-install or manually depending on this version.

Let me know if you have any trouble :)

**Comment by morugetsm** [comment]
I works well. Thank you so much!!

<img width="1728" alt="스크린샷 2022-11-21 오후 5 54 08" src="https://user-images.githubusercontent.com/90916540/203007429-64278646-8732-4aa5-af65-133b971b5175.png">
